web/third/bootstrap-typeahead/typeahead.css

```diff
@@ -83,8 +68,4 @@
 }
 .typeahead {
   z-index: 1051;
-  margin-top: 2px;
-  -webkit-border-radius: 4px;
-  -moz-border-radius: 4px;
-  border-radius: 4px;
```

Comment:
This file is probably ripe for being eliminated; with the remaining rules that actually do something moved to `popovers.css`;  seems like it has a lot of garbage that's overridden or done by modern bundlers.

Reply:
Yeah, I can do that in the next PR with the class renames.